client/cmd/internal/plan/directory.go

```diff
@@ -0,0 +1,49 @@
+package plan
+
+type DirectoryParser interface {
+	ValidDirectory(directory string) bool
+	ParseDirectory(directory string) string
+}
+
+func GetValidJobDirectory(directories []string) []string {
+	var (
+		jobDirectories []string
+		jobParser      = new(JobPlan)
+	)
+
+	for i := range directories {
```

Comment:
Can we use value instead of index ?
eg `for _, dir := range directories {`

Reply:
updated https://github.com/goto/optimus/pull/228/commits/5d58f748810ebf732a569c26c7eed09daa94b3c2 @sbchaos